container/apptabs.go

```diff
@@ -56,7 +56,7 @@ func NewAppTabs(items ...*TabItem) *AppTabs {
 		// Current is first tab item
 		tabs.current = 0
 	}
-	tabs.ExtendBaseWidget(tabs)
+	tabs.BaseWidget.ExtendBaseWidget(tabs)
```

Comment:
As the Deprecated ExtendBaseWidget is not a noop, are these changes needed?

Reply:
With it this change you have deprecation warnings... that seemed undesirable

Reply:
Ah fair enough